ci/orion.sh

```diff
@@ -0,0 +1,10 @@
+GDAS_CI_ROOT=/work2/noaa/stmp/cmartin/CI/GDASApp
```

Comment:
I couldn't find where this file is sourced?

Reply:
Line 37 of driver.sh